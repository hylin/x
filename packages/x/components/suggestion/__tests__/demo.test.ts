import demoTest from '../../../tests/shared/demoTest';

demoTest('suggestion', { testRootProps: false });
