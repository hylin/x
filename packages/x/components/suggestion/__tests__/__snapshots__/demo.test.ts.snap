// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders components/suggestion/demo/basic.tsx correctly 1`] = `
<div
  class="ant-suggestion ant-suggestion-wrapper"
>
  <div
    class="ant-sender"
  >
    <div
      class="ant-sender-content"
    >
      <textarea
        class="ant-input ant-input-borderless ant-sender-input"
        placeholder="输入 / 获取建议"
      />
      <div
        class="ant-sender-actions-list"
      >
        <div
          class="ant-sender-actions-list-presets ant-flex"
        >
          <button
            class="ant-btn ant-btn-circle ant-btn-primary ant-btn-color-primary ant-btn-variant-solid ant-btn-icon-only ant-sender-actions-btn ant-sender-actions-btn-disabled"
            type="button"
          >
            <span
              class="ant-btn-icon"
            >
              <span
                aria-label="arrow-up"
                class="anticon anticon-arrow-up"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="arrow-up"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                  />
                </svg>
              </span>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders components/suggestion/demo/block.tsx correctly 1`] = `
<div
  class="ant-suggestion ant-suggestion-wrapper"
>
  <div
    class="ant-sender"
  >
    <div
      class="ant-sender-content"
    >
      <textarea
        class="ant-input ant-input-borderless ant-sender-input"
        placeholder="输入 / 获取建议"
      />
      <div
        class="ant-sender-actions-list"
      >
        <div
          class="ant-sender-actions-list-presets ant-flex"
        >
          <button
            class="ant-btn ant-btn-circle ant-btn-primary ant-btn-color-primary ant-btn-variant-solid ant-btn-icon-only ant-sender-actions-btn ant-sender-actions-btn-disabled"
            type="button"
          >
            <span
              class="ant-btn-icon"
            >
              <span
                aria-label="arrow-up"
                class="anticon anticon-arrow-up"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="arrow-up"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                  />
                </svg>
              </span>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders components/suggestion/demo/trigger.tsx correctly 1`] = `
<div
  class="ant-suggestion ant-suggestion-wrapper"
>
  <div
    class="ant-select ant-select-outlined ant-select-multiple ant-select-show-arrow ant-select-show-search"
    style="width:100%"
  >
    <div
      class="ant-select-selector"
    >
      <span
        class="ant-select-selection-wrap"
      >
        <div
          class="ant-select-selection-overflow"
        >
          <div
            class="ant-select-selection-overflow-item ant-select-selection-overflow-item-suffix"
            style="opacity:1"
          >
            <div
              class="ant-select-selection-search"
              style="width:0"
            >
              <input
                aria-autocomplete="list"
                aria-controls="undefined_list"
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-owns="undefined_list"
                autocomplete="off"
                class="ant-select-selection-search-input"
                role="combobox"
                type="search"
                value=""
              />
              <span
                aria-hidden="true"
                class="ant-select-selection-search-mirror"
              >
              </span>
            </div>
          </div>
        </div>
        <span
          class="ant-select-selection-placeholder"
        >
          可任意输入 / 与 # 多次获取建议
        </span>
      </span>
    </div>
    <span
      aria-hidden="true"
      class="ant-select-arrow"
      style="user-select:none;-webkit-user-select:none"
      unselectable="on"
    >
      <span
        aria-label="down"
        class="anticon anticon-down ant-select-suffix"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="down"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
          />
        </svg>
      </span>
    </span>
  </div>
</div>
`;
